Sources/StreamChatSwiftUI/ChatChannel/MessageList/Polls/PollAttachmentViewModel.swift

```diff
@@ -62,7 +62,6 @@ public class PollAttachmentViewModel: ObservableObject, PollControllerDelegate {
         
     /// If true, an action sheet is shown for closing the poll, otherwise hidden.
     @Published public var endVoteConfirmationShown = false
-    @Published public var errorShown = false
```

Comment:
This is a breaking change, but not sure if anyone uses it tbh, wdyt?

Reply:
Pretty sure no-one uses it but I'll make it as deprecated.